### PR TITLE
Few small fixes

### PR DESCRIPTION
**Pull request by mzylowski**
Any suggestions? ;]

### CODE REVIEW

**Comment by mzylowski** [comment]
There are no comments, so merged ;]

File: lib/languages/pl.php [hunk 1]
```diff
@@ -428,7 +428,7 @@
     'newest_caches' => 'Najnowsze skrzynki',
     'total_of_active_caches' => 'Liczba aktywnych skrzynek',
     'total_of_caches' => 'Liczba skrzynek',
-    'total_moved' => 'Liczba przeniesionych skrzynek',
+    'total_moved' => 'Liczba transferów skrzynek mobilnych',
```
Comment (kojoty) [inline lib/languages/pl.php]: @mzylowski Have you checked how it sounds in other languages - maybe the translations should be also fixed?

Comment (mzylowski) [inline lib/languages/pl.php]: It looked like polish translation problem, but you are right.
In english it is: 'total_moved' => 'Number of moved caches',

Comment (kojoty) [inline lib/languages/pl.php]: zastanawiam się też, czy lepiej nie brzmiało by: "liczba przeniesień skrzynek mobilnych"

**Pull request merged**